src/System.Collections.Concurrent/tests/Performance/Perf.ConcurrentDictionary.cs

```diff
@@ -111,5 +111,56 @@ public void GetOrAdd(bool exists)
                 }
             }
         }
+
+        private static bool s_isEmptyResult;
+
+        [Benchmark(InnerIterationCount = 1_000_000)]
+        [InlineData(false)]
+        [InlineData(true)]
+        public void IsEmpty(bool isEmpty)
+        {
+            var d = new ConcurrentDictionary<long, long>();
+            if (!isEmpty)
+            {
+                d.TryAdd(42, 84);
+            }
+
+            s_isEmptyResult = true;
+            foreach (BenchmarkIteration iteration in Benchmark.Iterations)
+            {
+                long size = Benchmark.InnerIterationCount;
+                using (iteration.StartMeasurement())
+                {
+                    for (int i = 0; i < size; i++)
+                    {
+                        s_isEmptyResult &= d.IsEmpty;
```

Comment:
s_isEmptyDummy maybe? Since we don't use results of these

Reply:
Added NotUsed suffix.